wagtail/admin/templates/wagtailadmin/pages/page_listing_header.html

```diff
@@ -0,0 +1,34 @@
+{% extends 'wagtailadmin/shared/headers/slim_header.html' %}
```

Comment:
This may have been discussed in stand-up but I think the opening closing tags have been merged now, so we could move away from the `extends` approach here.